UnityProject/Assets/Scripts/Objects/StatusDisplay.cs

```diff
@@ -58,6 +58,19 @@ public enum MountedMonitorState
 		private bool multiMaster = true;
 		public bool MultiMaster => multiMaster;
 
+		private AccessRestrictions accessRestrictions;
+		public AccessRestrictions AccessRestrictions
+		{
+			get
+			{
+				if (!accessRestrictions)
```

Comment:
explictly compare to null instead ``accessRestrictions == null`` . This is mostly so new contributors understand what is happening here.